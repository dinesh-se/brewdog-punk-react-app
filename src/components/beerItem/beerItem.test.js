import React from 'react';
import { shallow } from 'enzyme';
import BeerItem from './beerItem';

test('should render', () => {
  const beer = {
    name: 'Beer 1',
    description: 'Beer description',
    image_url: 'image/png'
  };
  const wrapper = shallow(<BeerItem beer={beer} />);

  expect(wrapper).toMatchSnapshot();
  expect(wrapper.find('.beer-item-name').text()).toBe('Beer 1');
  expect(wrapper.find('.beer-item-desc').text()).toBe('Beer description');
});