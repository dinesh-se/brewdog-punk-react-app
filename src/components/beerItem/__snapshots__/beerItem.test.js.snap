// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="beer-item-container"
>
  <div
    className="flex-container"
  >
    <div
      className="image-container beer-item"
    >
      <img
        className="beer-image beer-item"
        src="image/png"
      />
    </div>
    <div
      className="beer-information"
    >
      <h3
        className="beer-item-name"
      >
        Beer 1
      </h3>
      <p
        className="beer-item-desc"
      >
        Beer description
      </p>
    </div>
  </div>
</div>
`;
